report/distributionList.go

```diff
@@ -0,0 +1,52 @@
+package report
+
+import (
+	"io/ioutil"
+
+	"github.com/davecgh/go-spew/spew"
+	"gopkg.in/yaml.v2"
+)
+
+// DistributionFormat is for unmarshiling a email distributionList file
+type DistributionFormat struct {
+	Severity SeverityDistribution `yaml:"severity"`
+}
+
+// SeverityDistribution list of emails ordered by severity
+type SeverityDistribution struct {
+	Debug []string `yaml:"debug,omitempty"`
+	Info  []string `yaml:"info,omitempty"`
+	Warn  []string `yaml:"warn,omitempty"`
+	Error []string `yaml:"error,omitempty"`
+	Fatal []string `yaml:"fatal,omitempty"`
+}
+
```

Comment:
Can SeverityDistribution just be an inline struct of DistributionFormat, or is it used independently elsewhere?  I think that would be a bit cleaner if possible.